src/org/oidc/common/OidcServiceError.java

```diff
@@ -0,0 +1,7 @@
+package org.oidc.common;
+
+public class OidcServiceError extends Exception{
```

Comment:
Rename it as OidcServiceException. That's the general convention.

In Java Exception and Error have very different meaning.